### PR TITLE
Reference management models language

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
Add language article

### CODE REVIEW

File: Reference/Management/Models/Language.md [hunk 1]
```diff
@@ -22,6 +19,61 @@ All samples in this document will require the following using statement:
...
+Gets or sets a value indicating whether the language is the default language.
+
+```csharp
+var language = new Language("en-US");
+return language.IsDefault;
+```
+
+### .IsMandatory
+
+Gets or sets the identifier of a fallback language. The fallback language can be used in multi-lingual scenarios, to help define fallback strategies when a value does not exist for a requested language.
```
Comment (busrasengul) [inline Reference/Management/Models/Language.md:65]: I think this sentences is the same with .FallbackLanguageId was it intentional?

File: Reference/Management/Models/Language.md [hunk 2]
```diff
@@ -22,6 +19,61 @@ All samples in this document will require the following using statement:
...
+Gets or sets the culture name of the language.
+
+```csharp
+var language = new Language("en-US");
+return language.CultureName;
+```
+
+### .FallbackLanguageId
+
+Gets or sets the identifier of a fallback language. The fallback language can be used in multi-lingual scenarios, to help  define fallback strategies when a value does not exist for a requested language.
```
Comment (busrasengul) [inline Reference/Management/Models/Language.md:48]:
```suggestion
Gets or sets the identifier of a fallback language. The fallback language can be used in multi-lingual scenarios, to help define fallback strategies when a value does not exist for a requested language.
```

**Review by busrasengul — CHANGES_REQUESTED** [review]
Hey @erikjanwestendorp 
I've add little comments would you mind checking them?
Thank you :)

File: Reference/Management/Models/Language.md (hunk 1 above)
Comment (erikjanwestendorp) [reply Reference/Management/Models/Language.md]: @busrasengul  Yes your right. I updated the description.

**Comment by erikjanwestendorp** [comment]
@busrasengul Thanks foy your review. Just updated the PR 😄

**Comment by busrasengul** [comment]
Thanks for updating them @erikjanwestendorp 
Looks good to me!
Merging it now :)

**Pull request merged**